### PR TITLE
fix(frontend): Update TFX context naming pattern for TFX 1.2.0+.

### PR DESCRIPTION
**Pull request by zijianjoy**
**Description of your changes:**

Reference: https://github.com/kubeflow/pipelines/issues/6138#issuecomment-898190223

It is required for TFX version to be > 1.2.0 starting with this change.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Comment by zijianjoy** [comment]
cc @jiyongjung0

File: frontend/src/lib/OutputArtifactLoader.ts [hunk 1]
```diff
@@ -272,6 +272,7 @@ export class OutputArtifactLoader {
 
     // Since artifact types don't change per run, this can be optimized further so
     // that we don't fetch them on every page load.
+    console.log('enter TFX for execution ' + execution.getId() + ' namespace ' + namespace);
```
Comment (Bobgy) [inline frontend/src/lib/OutputArtifactLoader.ts:275]: nit: clean up

File: frontend/src/mlmd/MlmdUtils.ts [hunk 2]
```diff
@@ -70,17 +70,9 @@ async function getContext({ type, name }: { type: string; name: string }): Promi
...
-  const pipelineName = argoWorkflowName
-    .split('-')
-    .slice(0, -1)
-    .join('_');
-  const runID = argoWorkflowName;
-  // An example run context name is parameterized_tfx_oss.parameterized-tfx-oss-4rq5v.
-  const tfxRunContextName = `${pipelineName}.${runID}`;
-  return await getContext({ name: tfxRunContextName, type: 'run' });
+  // Context: https://github.com/kubeflow/pipelines/issues/6138
+  // Require TFX version to be > 1.2.0.
```
Comment (Bobgy) [inline frontend/src/mlmd/MlmdUtils.ts:74]: nit: >=1.2.0

File: frontend/src/pages/RunDetails.tsx [hunk 3]
```diff
@@ -267,6 +267,7 @@ class RunDetails extends Page<RunDetailsInternalProps, RunDetailsState> {
       workflow,
       selectedNodeId,
     );
+    console.log('selectedNodeId ' + selectedNodeId);
```
Comment (Bobgy) [inline frontend/src/pages/RunDetails.tsx:270]: nit: clean up

**Pull request closed without merging**